data/projects/integrate-pyflakes-AST.md

```diff
@@ -0,0 +1,15 @@
+Integrate the [`pyflakes`](https://pypi.python.org/pypi/pyflakes)-enhanced [AST](https://en.wikipedia.org/wiki/Abstract_syntax_tree) into coala so that bears can access it by creating a meta bear like [AnnotationBear](https://github.com/coala/coala-bears/blob/1892b64722f1c10bd2a29611af4dcc18ad76af3a/bears/general/AnnotationBear.py) and [ClangBear](https://github.com/coala/coala-bears/blob/27a06d19076633a4e796472cf6d6ad55c27045c5/bears/c_languages/ClangBear.py), i.e PyflakesASTBear. Create two bears that wrap the flake8 AST plugins [`flake8-future-import`](https://github.com/xZise/flake8-future-import) and [`flake8-string-format`](https://github.com/xZise/flake8-string-format) using the PyflakesASTBear. The flake8 plugins needed to installed can be added to a dynamically created venv. Check out [this issue](https://gitlab.com/coala/package_manager/issues/25).
```

Comment:
This needs linebreaks and also some structure. Give it an introduction part and a "this is the idea what to do" part